tools/profiler/profiler.cpp

```diff
@@ -191,7 +191,7 @@ class ModelProfiler {
         forward_lats.push_back(
             std::chrono::duration_cast<duration>(end - beg).count());
       }
-      auto memory_usage = model->get_peak_memory_usage();
+      auto memory_usage = model->GetPeakMemoryUsage();
       CHECK_GE(memory_usage, 0) << "get_peak_memory_usage not implemented";
```

Comment:
Now that `GetPeakMemoryUsage` returns uint, the check always holds. Plus we have `LOG(FATAL)` as guard, let's just remove this line.

Reply:
Make sense.